.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
+sudo: false
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.9
+      - g++-4.9
+env: CXX=g++-4.9 CC=gcc-4.9
```

Comment:
the package ynodejs_gcc, which we use to build the ynodejs_core, has gcc version of 4.7.2. Will the 4.9 generated binaries work fine?